src/pages/LandingPage.tsx

```diff
@@ -14,16 +14,22 @@ export const LandingPage: React.FC = () => {
         <Row>
           <Col>
             <img className={styles.LandingPageFullWidthImagesContainer} src={landingPage1} />
-            <p className={styles.LandingPageOverlayText}>
-              <div className={styles.UnderlineBlockWrapper}>
-                <strong>Millions</strong>
-                <div className={styles.UnderlineBlock}></div>
-              </div>{' '}
-              of deaths from COVID-19 are underreported.
+            <div className={styles.LandingPageOverlayText}>
+              <p className={styles.OverlayTextMargin}>
+                <div className={styles.UnderlineBlockWrapper}>
+                  <strong>Millions</strong>
+                  <div className={styles.UnderlineBlock}></div>
+                </div>{' '}
+                of deaths from COVID-19 are underreported.
+              </p>
+              <div className={styles.LabDescription}>
+                Millions of deaths from COVID-19 are underreported. Millions of deaths from COVID-19
+                are underreported.
```

Comment:
Move into content file.  Update styling to match wireframes:

https://app.zeplin.io/project/624df3570bcb7b159718ab02/screen/62543bf4bc4e436bf126c489